content/guides/embed/ui-elements/explorer-metadata.md

```diff
@@ -0,0 +1,245 @@
+
+    // Get ENTERPRISE_ID from Developer Console (app's general settings)
+    const EID = "<ENTERPRISE_ID>";
+
+    // Get templatekey from Admin Console (Content -> Metadata -> check url for ID)
+    const templateName = "<METADATA_TEMPLATE_NAME>";
+
+    // Define metadata source
+    // Example: enterprise_123456789.metadatatemplate
+    const metadataSource = `enterprise_${EID}.${templateName}`;
+    const metadataSourceFieldName = `metadata.${metadataSource}`;
+    
+    const metadataQuery = {
+        from: metadataSource,
+
+        // Filter items in the folder by existing metadata key
+        query: "key = :arg1",
+
+        // Display items with value
+        query_params: { arg1: "value" },
+
+        // Define the ancestor folder ID
+        ancestor_folder_id: 0,
+
+        // Define which other metadata fields you'd like to display
+        fields: [
+            `${metadataSourceFieldName}.name`,
+            `${metadataSourceFieldName}.last_contacted_at`,
+            `${metadataSourceFieldName}.industry`,
+            `${metadataSourceFieldName}.role`,
+        ],
+    };
+
+    // The metadata fields/columns to view - must be valid field names from the metadata template
+    const fieldsToShow = [
+        // Determine if the user can edit the metadata directly from Content Explorer component
+        { key: `${metadataSourceFieldName}.name`, canEdit: false },
+
+        // Determine label alias on metadata column with displayName prop
+        { key: `${metadataSourceFieldName}.industry`, canEdit: false, displayName: "alias" },
+        { key: `${metadataSourceFieldName}.last_contacted_at`, canEdit: true },
+        { key: `${metadataSourceFieldName}.role`, canEdit: true },
+    ];
+
+    // defaultView - a required prop to paint the metadata view.
+    // If not provided, you'll get regular folder view.
+    const defaultView = "metadata";
+
+    return (
+        <IntlProvider locale="en">
+            <div className="App">
+                <header className="App-header">
+                    <h2>Metadata view in Content Explorer</h2>
+                </header>
+                <section>
+                    <div className="metadata-based-view">
+                        <ContentExplorer
+                            rootFolderId={rootFolderID}
+                            token={token}
+                            metadataQuery={metadataQuery}
+                            fieldsToShow={fieldsToShow}
+                            defaultView={defaultView}
+                        />
+                    </div>
+                </section>
+            </div>
+        </IntlProvider>
+    );
+}
+
+export default App;
+```
+
+## Key
+
+To decide which fields to show, the metadata Content Explorer uses metadata
+[field keys][field-key], not the [display names][display-name]. You can see the
+display names in the Admin Console and user view, but you can obtain the field
+keys through the API.
```

Comment:
Add link to API endpoint for getting the metadata template by ID

Reply:
I added the link as a related endpoint - I don't see where inline in this paragraph it could be added.